src/models/m_identity.erl

```diff
@@ -542,22 +543,34 @@ get_rsc(Id, Type, Context) ->
     z_db:assoc_row("select * from identity where rsc_id = $1 and type = $2", [Id, Type], Context).
 
 
-%% @doc Hash a password, using sha1 and a salt
+%% @doc Hash a password, using bcrypt
 %% @spec hash(Password) -> tuple()
 hash(Pw) ->
-    Salt = binary_to_list(z_ids:id(10)),
-    Hash = crypto:hash(sha, [Salt,Pw]),
-    {hash, Salt, Hash}.
+    {bcrypt, erlpass:hash(Pw)}.
 
 
+%% @doc Hash a password, with bcrypt
+%% @spec bcrypt_hash(Password) -> tuple
+
 %% @doc Compare if a password is the same as a hash.
 %% @spec hash_is_equal(Password, Hash) -> bool()
+hash_is_equal(Pw, {bcrypt, Hash}) ->
```

Comment:
BTW, when I run the tests locally I see a lot of warnings. I'm not sure if this works or not.

```
(zotonic001_testsandbox@Edo)1> 10:07:52.494 [info] No action enabled for "growl"
(zotonic001_testsandbox@Edo)1> 10:07:52.494 [error] [testsandboxdb] Could not start mod_authentication: Missing dependencies: [acl]
(zotonic001_testsandbox@Edo)1> 10:07:52.494 [info] No action enabled for "growl"
(zotonic001_testsandbox@Edo)1> 10:07:52.494 [error] [testsandboxdb] Could not start mod_admin: Missing dependencies: [authentication]
(zotonic001_testsandbox@Edo)1> 10:07:52.494 [info] No action enabled for "growl"
(zotonic001_testsandbox@Edo)1> 10:07:52.494 [error] [testsandboxdb] Could not start mod_admin_identity: Missing dependencies: [admin]
(zotonic001_testsandbox@Edo)1> 10:07:52.494 [info] No action enabled for "growl"
(zotonic001_testsandbox@Edo)1> 10:07:52.494 [error] [testsandboxdb] Could not start mod_admin_category: Missing dependencies: [admin]
(zotonic001_testsandbox@Edo)1> 10:07:52.495 [info] No action enabled for "growl"
(zotonic001_testsandbox@Edo)1> 10:07:52.495 [error] [testsandboxdb] Could not start mod_admin_predicate: Missing dependencies: [admin]
(zotonic001_testsandbox@Edo)1> 10:07:52.495 [info] No action enabled for "growl"
(zotonic001_testsandbox@Edo)1> 10:07:52.495 [error] [testsandboxdb] Could not start mod_admin_modules: Missing dependencies: [admin]
(zotonic001_testsandbox@Edo)1> 10:07:52.495 [info] No action enabled for "growl"
(zotonic001_testsandbox@Edo)1> 10:07:52.495 [error] [testsandboxdb] Could not start mod_admin_config: Missing dependencies: [admin]
```

And in the end...

```
(zotonic001_testsandbox@Edo)1>   All 23 tests passed.
(zotonic001_testsandbox@Edo)1> Terminating due to shutdown
(zotonic001_testsandbox@Edo)1> 
```

Is that normal?